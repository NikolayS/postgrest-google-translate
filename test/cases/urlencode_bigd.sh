#!/bin/bash

psql -U postgres test -c "select google_translate.urlencode('big â€” dash');"
