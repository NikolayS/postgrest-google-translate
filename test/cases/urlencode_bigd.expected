       urlencode        
------------------------
 big%20%E2%80%94%20dash
(1 row)

